### ISSUE TITLE
[Time series Forecasting] Continuous Ranked Probablity Score (CRPS) loss for probablity network ouput type

### ISSUE THREAD

**Issue by dengdifan** [automl/Auto-PyTorch#479]
We could consider implementing CPRS an alternative training loss for the probabilistic network output type.
INFO: https://www.lokad.com/continuous-ranked-probability-score

**Comment by omeurer** [comment]
Hey @dengdifan ! 
Within a university project, my team and I would like to contribute to this issue. If you have any more detailed specifications or hints, we would be glad to get them from you. 
Thanks for your work !

**Comment by dengdifan** [comment]
Hi @omeurer, 
sorry for the late reply, I am currently working on some other projects these days.

Basically, you could check if you can add a new loss type under this py module: https://github.com/automl/Auto-PyTorch/blob/master/autoPyTorch/pipeline/components/training/losses.py

Then you should attach the new loss to this dict: https://github.com/automl/Auto-PyTorch/blob/master/autoPyTorch/pipeline/components/training/losses.py#L128

Finally, some constraints might be required for this function: https://github.com/automl/Auto-PyTorch/blob/master/autoPyTorch/pipeline/time_series_forecasting.py#L207

if you have any further questions, please let me know.
